taprpc/mintrpc/mint.proto

```diff
@@ -16,6 +16,19 @@ service Mint {
     */
     rpc MintAsset (MintAssetRequest) returns (MintAssetResponse);
 
+    /* tapcli `assets mint fund`
+    FundBatch will attempt to fund the current pending batch with a genesis
+    input, or create a new funded batch if no batch exists yet.
+    */
+    rpc FundBatch (FundBatchRequest) returns (FundBatchResponse);
+
+    /* tapcli `assets mint seal`
+    SealBatch will attempt to seal the current pending batch by creating and
+    validating asset group witness for all assets in the batch. If a witness
+    is not provided, a signature will be derived to serve as the witness.
```

Comment:
Additional comment here could say "Calling this RPC method is only required if any of the assets in the batch have a custom group key that needs to be signed with externally. Otherwise FinalizeBatch can be called directly."